### ISSUE TITLE
How to use/test development version without installing?

### ISSUE THREAD

**Issue by gmaclennan** [developmentseed/landsat-util#126]
I am a Python newbie, but I want to play around with landsat-util, for example projecting into something other than eps:32718. How would I use the tool directly from the cloned github repo? I have also tried manually installing from the repo with `make install` but I get an error:

```sh
Processing dependencies for landsat-util==0.9.1
error: six 1.9.0 is installed but six==1.8.0 is required by set(['polyline'])
make: *** [install] Error 1
```

**Comment by scisco** [comment]
@gmaclennan after cloning the repo, if you run `pip install -r requirements/dev.txt` you should get all the dependencies installed. Then you can directly interact with the source code.

For example to run a command from the cloned repo just run `python landsat/landsat.py -h`

**Comment by jacquestardie** [comment]
@gmaclennan for reprojection, you may want to take a look at [rasterio](https://github.com/mapbox/rasterio/blob/master/docs/cli.rst#edit-info) — another Python library with some nice features for manipulating rasters.

**Comment by gmaclennan** [comment]
Thank you for this @scisco, this is just what I needed to get it working. To avoid re-projecting twice and any potential artifacts I replaced the epsg code in [image.py#L69-L70](https://github.com/developmentseed/landsat-util/blob/develop/landsat/image.py#L69-L70) with the projection I want, and it works just great.

I did have an error when running this command: `landsat/landsat.py process --pansharpen --clip=-77.5,-1.5,-77,-1 LC80090612015251LGN00.tar.bz`

```sh
Traceback (most recent call last):
  File "...landsat-util/landsat/landsat.py", line 458, in <module>
    __main__()
  File "...landsat-util/landsat/landsat.py", line 449, in __main__
    if args.json:
AttributeError: 'Namespace' object has no attribute 'json'
```

I just commented out the lines to get it working for now.
